app/code/Magento/Paypal/Controller/Express/AbstractExpress/PlaceOrder.php

```diff
@@ -49,8 +48,9 @@ public function __construct(
         \Magento\Framework\Url\Helper\Data $urlHelper,
         \Magento\Customer\Model\Url $customerUrl,
         \Magento\Checkout\Api\AgreementsValidatorInterface $agreementValidator,
-        \Magento\Sales\Api\PaymentFailuresInterface $paymentFailures = null
```

Comment:
Unfortunately, in terms of backward compatibility, we may not remove any dependencies even if we don't use them anymore.
Could you please restore this property `$paymentFailures` and add it back to the constructor.